src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -1579,4 +1694,99 @@ public with sharing class ADDR_Addresses_TEST {
             system.assertEquals(!fSeasonal, addr.Default_Address__c);
         }        
     }   
+
+    // Helpers
+    ////////////
+
+    /*********************************************************************************************************
+    * @description Populates Contact's Mailing Address from the Address
+    * @param c Contact
+    * @param sourceAddress Address
+    * @return Contact
+    **********************************************************************************************************/
+    static Contact populateContactAddress(Contact c, Address__c sourceAddress) {
```

Comment:
Interesting, I had never heard of that rule. It makes sense and does make it more clear, though, so I think you can leave it. I'll merge as is. Thanks!